### PR TITLE
Basic setup

### PR DESCRIPTION
**Pull request by zkr024**
- Set up the initial react app.
- Create a nav to move between the front and a secondary screen. 
- Create basic components.
- Basic style in the nav to pass the Linter test.

### CODE REVIEW

File: src/components/navBar/NavBar.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const NavBar = () => (
+  <div className="navContainer">
+    <h2>Bookstore CMS</h2>
+    <ul className="nav">
+      <li>
+        <Link to="/" className="link">BOOKS</Link>
+      </li>
+      <li>
+        <Link to="/categories" className="link">CATEGORIES</Link>
+      </li>
```
Comment (mateo951) [inline src/components/navBar/NavBar.js:14]: Has 2 `<Link>`s in the navbar which leads to the booklist and categories locations 🤓 Well done!

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import NotFound from './pages/NotFound';
+
+import NavBar from './components/navBar/NavBar';
+
+const App = () => (
+  <BrowserRouter>
+    <NavBar />
+    <Routes>
+      <Route path="/" element={<FrontScreen />} />
+      <Route path="/categories" element={<Categories />} />
```
Comment (mateo951) [inline src/App.js:14]: Has 2 `<Route>`s, one for the BookList component and the other for the Categories component 😎

File: src/components/books/AddBook.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        type="text"
+        placeholder="Author"
+        required
+      />
+      <button type="button">Add Book</button>
+    </form>
+  </>
+);
+
+export default AddBook;
```
Comment (mateo951) [inline src/components/books/AddBook.js:20]: New book form is rendered as expected 🤓

**Review by mateo951 — APPROVED** [review]
### Hi @zkr024, 

📝👌 **Your project highlights** 🔥😎 
- General Requirements ✔
- Nice PR 🔥

Your project is complete!  There is nothing else to say other than... it's time to merge it 🙌 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/books/AddBook.js (hunk 3 above)
Comment (zkr024) [reply src/components/books/AddBook.js:20]: @mateo951 Thank you for your feedback.

**Pull request merged**